src/tools/schema2base.ts

```diff
@@ -31,6 +31,13 @@ export interface ClassGenerator {
   generate(schemaHash: string, fieldCount: number): string;
 }
 
+export class NodeAndGenerator {
+  //await node.schema.hash(), fields.length)
```

Comment:
What's the go with all the commented out lines?

Reply:
Oops, this PR became a complete hot mess, and I missed a few comments 😞 Sorry about that.